execution/utility.sh

```diff
@@ -1396,8 +1396,17 @@ function syncFilesToBucket() {
       fi
     done
 
+    local target_url="s3://${bucket}/${prefix}${prefix:+/}"
+
     # Now synch with s3
-    aws --region ${region} s3 sync "${optional_arguments[@]}" "${tmp_dir}/" "s3://${bucket}/${prefix}${prefix:+/}"; return_status=$?
+    aws --region ${region} s3 sync "${optional_arguments[@]}" "${tmp_dir}/" "${target_url}"; return_status=$?
+    if [[ "${return_status}" -eq 0 ]]; then
+      readarray -t precompressed_files < <(find "${tmp_dir}" -type f -name "*-precompressed*" )
```

Comment:
I think adding it as a wild card across the whole file name is to broad. I'd suggest that its either a suffix or a prefix.

Reply:
Reworked to use a more unique filename prefix and also explicitly expect the enocding to be provided in the filename to allow arbitrary encoding values.